aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1664,7 +1661,7 @@ void int8_gemm(
       nullptr, // Heuristics don't seem to work for int8
 #endif
 #ifdef USE_ROCM
-      workspace.mutable_get(),
+      workspace.data_ptr(),
```

Comment:
```suggestion
      workspace.mutable_data_ptr(),
```

Reply:
etc... I think these are mostly for robust COW behavior of tensors, but no reason not to be explicit here.